src/post/PostSingleContent.scss

```diff
@@ -11,17 +11,21 @@
 
 .PostSingleContent__content,
 .PostSingleContent__replies .CommentBody {
-  font-size: 20px;
+  width: 820px;
```

Comment:
container class is defined by bootstrap. can we have another class for this width?